src/Services/Manager.vala

```diff
@@ -15,6 +15,11 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+[DBus (name="org.gnome.SettingsDaemon.Rfkill")]
+public interface Rfkill : Object {
```

Comment:
This should probably be namespaced to make sure we don't conflict with other indicators, for example if the wifi indicator uses RFKill
```suggestion
public interface BluetoothIndicator.Services.Rfkill : Object {
```